### PR TITLE
working pre label worker

### PR DESCRIPTION
**Pull request by mengting-bst**
this branch include a working pre-label worker using fast-rcnn.

### CODE REVIEW

File: aws/sqs.py [hunk 1]
```diff
@@ -5,18 +5,18 @@
 class BSTQueues(object):
```
Comment (guan-bst) [inline aws/sqs.py]: It may be necessary to seperate aws functions into something like "aws-py", just like aws-go, since they could be useful in later projects.

File: pre_labeler.py [hunk 2]
```diff
@@ -33,6 +33,11 @@ def start(worker):
     for i in range(0, NUMBER_OF_THREAD):
```
Comment (guan-bst) [inline pre_labeler.py]: how much faster can we get if NUMBER_OF_THREAD > 1?

File: fast_rcnn/fast_rcnn_worker.py [hunk 3]
```diff
@@ -1,21 +1,127 @@
...
+        if args.cuda > 0:
+            checkpoint = torch.load(args.path_to_model)
+        else:
+            checkpoint = torch.load(args.path_to_model, map_location=(lambda storage, loc: storage))
+        fasterRCNN.load_state_dict(checkpoint['model'])
+        if 'pooling_mode' in checkpoint.keys():
+            cfg.POOLING_MODE = checkpoint['pooling_mode']
+        print('load model successfully!')
+
+        cfg.USE_GPU_NMS = args.cuda
```
Comment (guan-bst) [inline fast_rcnn/fast_rcnn_worker.py]: if your EC2 instance has GPU, this args.cuda needs to be set. Just FYI.
